native/calendar/entry.react.js

```diff
@@ -392,19 +391,13 @@ class InternalEntry extends React.Component<Props, State> {
             iosActiveOpacity={opacity}
           >
             <View>
-              <View style={heightStyle} />
-              <Hyperlink
-                linkDefault={true}
-                linkStyle={linkStyle}
-                style={this.props.styles.textContainer}
+              <View style={[heightStyle, this.props.styles.textContainer]} />
+              <Text
+                style={[this.props.styles.text, textStyle]}
+                onLayout={this.onTextLayout}
               >
-                <Text
-                  style={[this.props.styles.text, textStyle]}
-                  onLayout={this.onTextLayout}
-                >
-                  {rawText}
-                </Text>
-              </Hyperlink>
+                <Markdown darkStyle={darkColor}>{rawText}</Markdown>
```

Comment:
Another issue I noticed in testing. If I add two line breaks after some text, the height calculation on `Markdown` gets broken. This indicates something wrong within the `Markdown` component itself

Reply:
Hmm, this issue seems to repro for messages. If I send a message `Test\n\nA` it turns into `TestA`. I think we tested this, so I suspect one of our later changes broke it. I'll try to investigate this as well...

Reply:
I finished the investigation and updated #67. Check out my comment there, and make sure to rebase this PR against the updated `feature/message_formatting` branch!